src/player.js

```diff
@@ -1706,6 +1721,18 @@ export default class Player extends FakeEventTarget {
         this._eventManager.listen(this, AdEventType.AD_BREAK_END, this._attachMediaSource);
         this._eventManager.listen(this, AdEventType.AD_ERROR, this._attachMediaSource);
       }
+      const rootElement = Utils.Dom.getElementBySelector(`#${this.config.targetId}`);
```

Comment:
Could be an issue when player is smaller than root element.
Maybe playkit-video-player could do the trick

Reply:
I tested it with a player of 50% - works fine